pkg/contracts/src/RegistryCommunity.sol

```diff
@@ -496,8 +504,11 @@ contract RegistryCommunity is ReentrancyGuard, AccessControl {
         emit BasisStakedAmountSet(_newAmount);
     }
 
-    function updateCommunityFee(uint256 _newCommunityFee) public {
+    function setCommunityFee(uint256 _newCommunityFee) public {
         onlyCouncilSafe();
+        if (_newCommunityFee > MAX_FEE) {
+            revert NewFeeGreaterThanMax();
+        }
```

Comment:
what reason behind to change the name?

Reply:
I liked it better but I realised if it breaks the subgraph and stuff we can remove

Reply:
i chekced not used yet, so safe to change.